src/config.ts

```diff
@@ -181,8 +182,28 @@ export const STARTUP_IDPS_METADATA: Record<string, string> | undefined = pipe(
   O.getOrElseW(() => undefined)
 );
 
-export const CLIENT_ERROR_REDIRECTION_URL =
-  process.env.CLIENT_ERROR_REDIRECTION_URL || "/error.html";
+export const CLIENT_ERROR_REDIRECTION_URL = pipe(
+  // this env variable is required and must be an absolute URL
+  process.env.CLIENT_ERROR_REDIRECTION_URL,
+  NonEmptyString.decode,
+  E.mapLeft((err) => {
+    throw new Error(
```

Comment:
The throw is useless if we have a `process.exit()` on the final `getOrElse`. Map only the error instead

Reply:
done in https://github.com/pagopa/io-backend/pull/1012/commits/3ada102f061d73a9a4fc9258f840af9f6d3e6b7b